### PR TITLE
Markus Doan-Le completed css-two-column-layout exercise

### PR DESCRIPTION
**Pull request by mdoanle**
Screen Shot 2022-08-27 at 6 13 48 PM

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, good job so far, but there's a lot of duplication in your code. Try generalizing your typography CSS to start, so instead of `.top-right > h1` or `.africa-image` etc, try just making a `.heading` and `.image` and put all of the common styles in them. You should be able to cut like 20 lines of code that way.

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+.container {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  justify-content: right;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:8]: Don't pollute your layout classes. You want to keep layout classes perfectly reusable, and augmenting their intended functionality immediately makes them less reusable. You should move this `justify-content` declaration to a separate class, and you can apply it in addition to the `row` class to any elements that need this behavior.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+.column-full {
+  flex-basis: 100%;
+  justify-content: center;
+  text-align: center;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:14]: Don't pollute your layout classes. `column-` classes should only include `flex-basis` or `width` and nothing else. Please apply these additional declarations to a separate class that you can add in addition to the `column` class

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  flex-basis: calc(200%/3);
+}
+
+#top-row {
+  margin-right: 5rem;
+}
+
+#flexbox-tl {
+  margin-top: 2rem;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:65]: You should avoid applying styling to `id`s, please shift these to classes.

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+#image-br {
+  margin-left: 4rem;
+  margin-bottom: 4rem;
+}
+
+#final-row {
+  margin-top: 5px;
+  margin-bottom: 3rem;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:115]: You should avoid applying styling to `id`s, please shift these to classes.

File: css-two-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+input {
+  padding: 10px 3rem;
+  background-color: white;
+  border: solid red;
+}
+
+[value] {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:123]: This selector is kind of confusing, I would just add a class to the submit button that you can apply this styling to.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**